src/components/addProgram/AddProgram.jsx

```diff
@@ -80,6 +99,25 @@ const AddProgram = (props) => {
           return stackObject;
         },
       );
+      newInternship.languageTypes = newInternship.languageTypes.map(
+        (language) => {
+          const languageObject = { language };
+          return languageObject;
+        },
+      );
+      // newInternship.users = newInternship.users.map(
+      //   (user) => {
+      //     const userObject = {...allUsersList.find((item) => {
+      //       if (user === item.userName){
+      //         return true
+      //       }
+      //       return false
+      //     })}
+      //     return userObject;
+      //   },
+      // );
```

Comment:
Please delete unused comments

Reply:
I put a stub so that instead of an array with users, an empty array would come. Since the back-end eliminates problems with user acceptance.

Reply:
I will need this code

Reply:
I don't like it. But ok. Leave comments